core/util/runPuppet.js

```diff
@@ -199,9 +204,20 @@ async function processScenarioView (scenario, variantOrScenarioLabelSafe, scenar
         backstopSelectorsExpMap: window._backstopSelectorsExpMap
       };
     })
-
-  return new Promise((resolve, reject) => {
-    resolve(delegateSelectors(
+  }
+  
+  await puppetCommands().catch(e => {
+    result = {
```

Comment:
Should this be a const?  It looks like this `result` is not the same `result` used above?  Or did I maybe read this wrong?

Reply:
it is actually the same result.. If the puppeteer commands fail the result is empty and this was the best I could come up with without refactoring too much.. 🛠

Reply:
Cool.  Thanks!